cmd/obslytics/export.go

```diff
@@ -66,20 +67,47 @@ func registerExport(m map[string]setupFunc, app *kingpin.Application) {
 			if err != nil {
 				return err
 			}
-			out, err := outfactory.Parse(outputCfg)
+			out, err := outfactory.Parse(logger, outputCfg)
 			if err != nil {
 				return err
 			}
 
-			w, err := os.Create(*outFile)
+			outFileName := *outFile
+
+			storageBucket := out.GetStorageBucket()
+			// If Object Storage is set up, store output file to a temp directory, then later upload it to object storage.
```

Comment:
Why is that?  We can directly write in object storage, no?

Reply:
We can, but that would mean the file is first written to the temp directory and then copied to the destination directory, would that be okay?

Reply:
Why? We can directly write to dest place, no?

Reply:
I couldn't figure that out

Reply:
The parquet writer writes directly to a file, I couldn't find a way to change it.

Reply:
I see, we can contribute some change or just do it temp file then, but it would be nice to encapsulate this inside output, WDYT? (:

Reply:
yeah that was the idea, when I kept the storageBucket inside the Output.
This way, in the future we could do the uploading while the output is being processed (i.e. being converted to Parquet)

Reply:
Updated, now we are using objstor to write to file